packages/animations/test/open_container_test.dart

```diff
@@ -1530,26 +1530,37 @@ void main() {
     @required Key nestedNavigatorKey,
     @required bool useRootNavigator,
   }) {
-    return MaterialApp(
-        key: appKey,
-        // a nested navigator
-        home: Navigator(
-            key: nestedNavigatorKey,
-            onGenerateRoute: (RouteSettings route) {
-              return MaterialPageRoute<dynamic>(
-                  settings: route,
-                  builder: (BuildContext context) {
-                    return Container(
-                        child: OpenContainer(
-                            useRootNavigator: useRootNavigator,
-                            closedBuilder: (BuildContext context, _) {
-                              return const Text('Closed');
-                            },
-                            openBuilder: (BuildContext context, _) {
-                              return const Text('Opened');
-                            }));
-                  });
-            }));
+    return Center(
+      child: SizedBox(
+        width: 100,
+        height: 100,
+        child: MaterialApp(
+            key: appKey,
+            // a nested navigator
+            home: Center(
+              child: SizedBox(
+                width: 50,
+                height: 50,
+                child: Navigator(
+                    key: nestedNavigatorKey,
+                    onGenerateRoute: (RouteSettings route) {
+                      return MaterialPageRoute<dynamic>(
+                          settings: route,
+                          builder: (BuildContext context) {
+                            return OpenContainer(
+                                useRootNavigator: useRootNavigator,
+                                closedBuilder: (BuildContext context, _) {
+                                  return const Text('Closed');
+                                },
+                                openBuilder: (BuildContext context, _) {
+                                  return const Text('Opened');
+                                });
+                          });
+                    }),
+              ),
+            )),
+      ),
+    );
```

Comment:
nit: use trailing commas throughout this block to help formatter

Reply:
OK, will fix in next commit